### PR TITLE
pt-BR translation

### PR DESCRIPTION
**Pull request by iraamaro**

### CODE REVIEW

File: .resources/strings.json [hunk 1]
```diff
@@ -195,46 +218,52 @@
 		"ja": "表示",
 		"es": "Vista",
 		"de": "Anzeigen"
+		"pt-BR": "Ver"
```
Comment (szTheory) [inline .resources/strings.json:221]: Not sure if this is some kind of special `:` character, it's highlighted in the pull request on Github. If so can you change it to a normal colon character before I merge this in?

File: .resources/strings.json [hunk 2]
```diff
@@ -243,104 +272,116 @@
...
 		"fr": "Fermer",
 		"pl": "Zamknij",
 		"ja": "閉じる",
 		"es": "Cerrar",
-		"de": "Schließen"
-	},
+		"en": "Schließen",
+		"pt-BR": "Fechar"
 
 	"menu.help.name": {
```
Comment (szTheory) [inline .resources/strings.json:341]: It's highlighting this colon as red too, maybe I pasted in a wrong character before? Can you change it to a normal colon character

File: .resources/strings.json [hunk 3]
```diff
@@ -243,104 +272,116 @@
...
 		"pl": "Zoom",
 		"ja": "ズーム",
 		"es": "Maximizar",
-		"de": "Zoomen"
+		"de": "Zoomen",
+		"pt-BR": "Ampliar"
 	},
 	"menu.window.front": {
-		"en": "Brint All to Front",
+		"en": "Bring All to Front",
```
Comment (szTheory) [inline .resources/strings.json:315]: Good catch!

File: .resources/strings.json [hunk 4]
```diff
@@ -72,38 +80,43 @@
 		"ja": "サービス",
 		"es": "Servicios",
 		"de": "Services"
+		"pt-BR": "Serviços"
```
Comment (szTheory) [inline .resources/strings.json:83]: This colon character is highlighted in the PR code for some reason can you change it to a normal colon character if it's some weird unicode version?

File: .resources/strings.json [hunk 5]
```diff
@@ -21,23 +23,26 @@
...
 	},
 	"table.header.exif-after": {
 		"en": "# Exif After",
 		"fr": "# Exif Après",
 		"pl": "# Exif po",
 		"ja": "# 処理後の Exif",
 		"es": "Exif después",
-		"de": "# Exif nachher"
+		"de": "# Exif nachher",
+		"pt-BR": "#Exif depois"
```
Comment (szTheory) [inline .resources/strings.json:45]: Needs a space after the `#`

**Comment by szTheory** [comment]
Awesome thanks, this looks great! I just left a few comments for changes before merging into master.

**Review by szTheory — CHANGES_REQUESTED** [review]

File: .resources/strings.json (hunk 2 above)
Comment (iraamaro) [reply .resources/strings.json:341]: All red highlights fields were a bug from my text editor lint, [micro](https://github.com/zyedidia/micro).

**Comment by szTheory** [comment]
@iraamaro  I've just released a new version v3.4.0 that includes your translation, as well as a bunch of other new features and fixes. Thanks again for your contribution! https://github.com/szTheory/exifcleaner/releases/tag/v3.4.0

**Pull request merged**